CMakeLists.txt

```diff
@@ -157,6 +157,18 @@ macro (sources_hook)
                                 PROPERTIES COMPILE_DEFINITIONS "${qm_defs}"
                                 COMPILE_OPTIONS "${qm_options}")
   endif()
+  # Set the path to the input docstrings.json file and the output .hpp file
+  set(PYTHON_DOCSTRINGS_FILE "${PROJECT_SOURCE_DIR}/python/cxx/docstrings.json")
+  set(PYTHON_DOCSTRINGS_GENERATED_HPP "${PROJECT_BINARY_DIR}/python/cxx/OpmCommonPythonDoc.hpp")
+  # Command to run the Python script
+  add_custom_command(
+      OUTPUT ${PYTHON_DOCSTRINGS_GENERATED_HPP}
+      COMMAND ${PYTHON_EXECUTABLE} ${PROJECT_SOURCE_DIR}/python/generate_docstring_hpp.py
+              ${PYTHON_DOCSTRINGS_FILE} ${PYTHON_DOCSTRINGS_GENERATED_HPP} OPMCOMMONPYTHONDOC_HPP "Opm::Common::DocStrings"
+      DEPENDS ${PYTHON_DOCSTRINGS_FILE}
+      COMMENT "Generating OpmCommonPythonDoc.hpp from JSON file"
+  )
+  list(INSERT opm-common_SOURCES 0 ${PYTHON_DOCSTRINGS_GENERATED_HPP})
```

Comment:
Does this run unconditionally&ndash;i.e., whether Python is found/enabled or not?

Reply:
I've changed this now, thanks!